styles.css

```diff
@@ -0,0 +1,3 @@
+h1 {
+  color: blue;
+}
```

Comment:
I have already finished all you mentioned. But I accidentally merged and the PR closed. How can I reopen it?

Reply:
I think I can't. Shall I create a new PR? Please help me.

Reply:
Sorry, I was away for some times
Are you able to fix it now or you still need help

Reply:
I corrected all you mentioned. My problem is with PR. Shall I create another PR? or Submit it with the closed PR? Because I merged it before looking at your review, my bad.